charts/backbeat-api/templates/deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "backbeat-api.fullname" . }}
+  labels:
+    app: {{ template "backbeat-api.name" . }}
+    chart: {{ template "backbeat-api.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "backbeat-api.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: backbeat-api
+              containerPort: 8900
+          env:
```

Comment:
Can you add the ```REMOTE_MANAGEMENT_DISABLE```orbit variable (just like S3) as it was added to Backbeat as well.

Reply:
https://github.com/scality/backbeat/blob/bf08d9600a7d62ab725ce7e13e789cfbdf2166ea/lib/management.js#L336